Signal/src/ViewControllers/ThreadSettings/OWSConversationSettingsViewController.m

```diff
@@ -37,10 +37,13 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
-@interface OWSConversationSettingsViewController () <ContactEditingDelegate, ContactsViewHelperDelegate>
+@interface OWSConversationSettingsViewController () <ContactEditingDelegate,
+    ContactsViewHelperDelegate,
+    ColorPickerDelegate>
 
 @property (nonatomic) TSThread *thread;
 @property (nonatomic) YapDatabaseConnection *uiDatabaseConnection;
+@property (nonatomic) YapDatabaseConnection *editingDatabaseConnection;
```

Comment:
Why declare this private property - isn't the method below sufficient?

Reply:
The method below is sufficient, but, and this is a small subjective style point, properties aren't just for wrapping ivars - I (and Apple) prefer to model idempotent getters as properties. 

e.g. if I wanted to access this connection elsewhere in the code I will probably write: `self.editingDatabaseConnection` not `[self editingDatabaseConnection]`

To me that's a sign that this should be a property. In objc they are of course absolutely equivalent, but Swift actually enforces this distinction (e.g. `self.editingDatabaseConnection()` vs `self.editingDatabaseConnection`).